doc/release-process.md

```diff
@@ -325,6 +327,8 @@ bitcoin.org (see below for bitcoin.org update instructions).
 
       - Create a [new GitHub release](https://github.com/bitcoin/bitcoin/releases/new) with a link to the archived release notes.
 
+      - Create an issue dedicated to testing the release and provide the link to it in the release announcements where useful
```

Comment:
I don't like having a single issue/thread for all issues with a release. It could be a meta issue, but all bug reports should go into their own issue.

Reply:
Thank you for reviewing. Prefer re-word (e.g. "meta issue" or "issue for release ACKs") or remove?

Reply:
Updated.